config/initializers/instantiation_listener.rb

```diff
@@ -1,10 +1,12 @@
-ActiveSupport::Notifications.subscribe('instantiation.active_record') do |name, start, finish, _id, payload|
-  logger = ActiveRecord::Base.logger
-  if logger.debug?
-    elapsed = finish - start
-    name = payload[:class_name]
-    count = payload[:record_count]
+unless ENV['HIDE_INSTANTIATION_DEBUG']
```

Comment:
I wonder  if it makes sense to have `ENV['MIQ_QUIET']` or `ENV['MIQ_VERBOSE']` and we can start putting similar loud things behind a single ENV variable to "opt-in" for quiet mode.

@Fryguy mentioned the connection pool logging and maybe both of these should be filtered out in dev mode with a QUIET mode or non-verbose mode.  I can see value in wanting to see the SQL queries but possibly not the `Inst` lines nor the connection pool checkin/checkout information.

Reply:
Yeah, I think a standard variable is a good idea.  👍

Reply:
`MIQ_NOT_SO_VERBOSE_BUT_STILL_DEBUG` 😂   Naming is hard

Reply:
`MIQ_SOME_PEOPLE_GET_BOTHERED_BY_CONNECTION_POOL_LOGGING_BUT_ARE_FINE_WITH_CONSTANT_OS_NOTIFICATIONS`

🤣 🥇